packages/vue/examples/__tests__/commits.spec.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import { setupPuppeteer } from './e2eUtils'
+
+describe('e2e: commits', () => {
+  const { page, click, count, text, isChecked } = setupPuppeteer()
+
+  async function testCommits(apiType: 'classic' | 'composition') {
+    const baseUrl = `file://${path.resolve(
+      __dirname,
+      `../${apiType}/commits.html`
+    )}`
+
+    await page().goto(baseUrl)
+    await page().waitForResponse(
+      'https://api.github.com/repos/vuejs/vue-next/commits?per_page=3&sha=master'
```

Comment:
maybe you should declare `url` as a static parameter, and move wrap waitForResponse to `e2eUtils.ts`